ui/tailwind.config.js

```diff
@@ -15,13 +15,15 @@ module.exports = {
   theme: {
     extend: {
       colors: {
+        primary: '#FFFFFF',
+        secondary: '#B2B2B2',
```

Comment:
Can we start our own "gray" theme  e.g. `text-gray-300` vs having one-off names

https://tailwindcss.com/docs/customizing-colors